cfgov/unprocessed/apps/mmt-my-money-calendar/js/stores/models/categories.js

```diff
@@ -83,7 +83,7 @@ export const Categories = new CategoryTree({
     salary: {
       name: 'Job',
       icon: icons.job,
-      description: 'Income from employment.',
+      /* description: 'Income from employment.', */
```

Comment:
Any reason for leaving this as a comment?

Reply:
I removed comment.